### PR TITLE
Class room edit and delete

### PR DESCRIPTION
**Pull request by Kathuria**
#6 

Added required feature of Classroom Edit and Delete.
Delete and Edit

### CODE REVIEW

**Comment by Kathuria** [comment]
@harshareddy794 , @mohit355 , @aavishkarmishra : Can you please review this

File: client/src/components/ClassCard/ClassCard.js [hunk 1]
```diff
@@ -103,7 +103,7 @@ export default function SimpleCard(props) {
             Class Code: {Class.code}
           </Typography>
         </div>
-        <DeleteIcon color='error' className={classes.delete} onClick={() => deleteClass(Class.code, Class.instructor?.email, setClasses)}/>
+        <DeleteIcon color='error' className={classes.delete} onClick={() => deleteClass(Class.code, Class.instructor?.email, setClasses, admin)}/>
```
Comment (aavishkarmishra) [inline client/src/components/ClassCard/ClassCard.js:106]: In this design delete icon can also work as leave option for students. Please add this feature.

**Review by aavishkarmishra — CHANGES_REQUESTED** [review]

File: client/src/components/ClassCard/ClassCard.js (hunk 1 above)
Comment (Kathuria) [reply client/src/components/ClassCard/ClassCard.js:106]: @aavishkarmishra : Yes adding as part of #28

Comment (Kathuria) [reply client/src/components/ClassCard/ClassCard.js:106]: @aavishkarmishra : Done, Please check new PR

**Pull request merged**